arm/Microsoft.Authorization/policySetDefinitions/managementGroup/deploy.bicep

```
@@ -1,7 +1,7 @@
 targetScope = 'managementGroup'
 
-@sys.description('Required. Specifies the name of the policy Set Definition (Initiative). Maximum length is 24 characters for management group scope.')
-@maxLength(24)
+@sys.description('Required. Specifies the name of the policy Set Definition (Initiative). Maximum length is 64 characters for management group scope.')
```

If I understand correctly it would be 64 max not only for mg scope but for all scopes. In that case, for consistency with others, I'd suggest:
```suggestion
@sys.description('Required. Specifies the name of the policy Set Definition (Initiative).')
```